backend/tests/conftest.py

```diff
@@ -1,29 +1,26 @@
-from datetime import datetime
+import datetime
 
 import pytest
 from fastapi.testclient import TestClient
-from sqlalchemy.orm import scoped_session
 
 from backend.database import Session
 from backend.database.models import Project, User
 from src.backend.entrypoint import app
 
 
 @pytest.fixture()
-def mock_user_id():
-    new_user = User(created_on=datetime.utcnow(), projects=[])
-    with scoped_session(Session)() as session:
+def user_id():
```

Comment:
OK but now we're not deleting at all.
Here's how to:

```py
@pytest.fixture()
def user_id():
    new_user = User(created_on=datetime.datetime.utcnow(), projects=[])
    with Session.begin() as session:
        session.add(new_user)
        session.flush()
        session.refresh(new_user)
        created_id = new_user.id
    yield created_id
    with Session.begin() as session:
        user = session.get(User, created_id)
        if user:
            session.delete(user)

...

@pytest.fixture()
def project_id(test_project_name, user_id):
    now = datetime.datetime.utcnow()
    new_project = Project(
        name=test_project_name, created_on=now, expire_on=None, files=[], archives=[]
    )
    with Session.begin() as session:
        user = session.get(User, user_id)
        if user:
            user.projects.append(new_project)
        session.add(new_project)
        session.flush()
        session.refresh(new_project)
        created_id = new_project.id
    yield created_id
    with Session.begin() as session:
        project = session.get(User, user_id)
        if project:
            session.delete(project)
```

Reply:
Ok